release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -297,16 +297,32 @@ For more information, see xref:../networking/metallb/metallb-configure-bgp-peers
 [id="ocp-4-14-networking-nmstate-vrf"]
 ==== Support for VRF specification in NMState's NodeNetworkConfigurationPolicy resource (Technology Preview)
 
-With this update, you can assoicate a Virtual Routing and Forwarding (VRF) instance with a network interface by using a `NodeNetworkConfigurationPolicy` custom resource. By associating a VRF instance with a network interface, you can support traffic isolation, independent routing decisions, and the logical separation of network resources. This is available as a Technology Preview feature.
+With this update, you can associate a Virtual Routing and Forwarding (VRF) instance with a network interface by using a `NodeNetworkConfigurationPolicy` custom resource. By associating a VRF instance with a network interface, you can support traffic isolation, independent routing decisions, and the logical separation of network resources. This is available as a Technology Preview feature.
 
 For more information, see xref:../networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.adoc#virt-example-host-vrf_k8s_nmstate-updating-node-network-config[Example: Network interface with a VRF instance node network configuration policy].
 
 [id="ocp-414-broadcom-bcm57504-support"]
 ==== Support for Broadcom BCM57504 is now GA
 
-Support for the Broadcom BCM57504 network interface controller is now available for the SR-IOV Network Operator. 
+Support for the Broadcom BCM57504 network interface controller is now available for the SR-IOV Network Operator.
 
-For more information, see xref:../networking/hardware_networks/about-sriov.adoc#supported-devices_about-sriov[Supported devices].
+For more information, see xref:../networking/hardware_networks/about-sriov.adoc#supported-devices_about-sriov
+
+[Supported devices].
+[id="ocp-4-14-creating-subinterface"]
+==== MAC-VLAN, IP-VLAN, and VLAN subinterface creation for pods
+
+With this release, the ability to create a MAC-VLAN, IP-VLAN and VLAN subinterface based on a master interface in a container namespace is generally available. This feature allows you to create the master interfaces as part of the pod network configuration in a separate network attachment definition, and base the VLAN, MACVLAN or IPVLAN on this interface without requiring the knowledge of the network configuration of the node. For more information, see xref:../networking/multiple_networks/configuring-additional-network.html#nw-about-configuring-master-interface-container_configuring-additional-network[About configuring the master interface in the container network namespace].
```

Comment:
`Master`'s fine here given context and existing use in published docs.

Reply:
Consider more user-focused language, e.g.:

> With this release, you can now create MAC-VLAN, IP-VLAN, and VLAN subinterfaces based on master namespaces in container namespaces.

Additionally, make sure to use serial commas.

> With this release, the ability to create a MAC-VLAN, IP-VLAN, and [or?] VLAN subinterface based on a master interface in a container namespace is generally available.